webots_ros2_msgs/CMakeLists.txt

```diff
@@ -32,10 +32,11 @@ set(msg_files
   "msg/PenInkProperties.msg"
 )
 set(srv_files
-  "srv/GetBool.srv"
   "srv/EmitterSendString.srv"
-  "srv/SpawnUrdfRobot.srv"
+  "srv/GetBool.srv"
+  "srv/SetString.srv"
   "srv/SpawnNodeFromString.srv"
+  "srv/SpawnUrdfRobot.srv"
```

Comment:
This is just to keep them sorted. It's actually just one line change. Should I revert?

Reply:
It's nice to have sorted lists, so I believe your change is good